dcos_installer/exhibitor_migrate.py

```diff
@@ -0,0 +1,160 @@
+"""
+exhibitor-migrate
+=================
+When a user is updating to DC/OS 1.9 from an older version we need to update several values in exhibitor to use the new
+filesystem paths for zookeeper state on disk.
+"""
+import argparse
+import copy
+import sys
+
+import requests
+from requests import RequestException
+
+DESCRIPTION_STATUS = """
+Will query exhibitor (at `localhost`) and check if the migration has been completed, in progress, or has not yet been
+performed.
+
+| exit code | State                                                                         |
+| --------- | ----------------------------------------------------------------------------- |
+| 0         | Exhibitor has new expected values and no rollout is in progress               |
+| 1         | IO Error when trying to connect to exhibitor (expected if run on an agent)    |
+| 2         | Error when reading from exhibitor (Any response other than 200)               |
+| 4         | Rolling update in progress                                                    |
+| 8         | At least one config value does not have the expected post migration value     |
+
+
+This script can be run on any of the DC/OS Master nodes by running (as root)
+
+```
+dcos-shell dcos-exhibitor-migrate-status
+```
+"""  # noqa
+
+DESCRIPTION_PERFORM = """
+Will attempt to update the exhibitor (at `localhost`) config and trigger a rolling update .
+
+| exit code | State                                                                                                                 |
+| --------- | --------------------------------------------------------------------------------------------------------------------- |
```

Comment:
nit: could look weird in terminals that aren't wide enough. That said not worth fixing to me currently.